android/app/src/main/res/layout/fragment_home.xml

```diff
@@ -3,24 +3,40 @@
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
+    android:layout_height="match_parent"
+    android:background="@drawable/bg_app"
+    android:padding="@dimen/default_margin_16">
+
+    <View
+        android:id="@+id/vHomeBackground"
+        android:layout_width="match_parent"
+        android:layout_height="0dp"
```

Comment:
Why don't we set match_parent and get rid of constraint top/bottom?

Reply:
Oh, it can. Thanks. 🙏🏻